js/viz.js

```diff
@@ -45,22 +49,44 @@ const viz = {
       simulation.nodes(this.nodes);
     }
 
+    this.registerSimulationForces(simulation);
+
+    simulation.alphaTarget(1);
+    for (let i = 0; i < 10; i++) {
+      simulation.tick();
+    }
```

Comment:
we can register event handler for `tick` as below:
`simulation.on('tick', () => doSomething);`
This wasn't behaving as expected, the graph was starting from 0,0 and not taking into account the simulationForces etc. So I resolved to having manual tick as a temporary solution.